kernel/will-it-scale.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2024 IBM
+# Author: Sachin Sant <sachinp@linux.ibm.com>
+#
+
+
+import os
+import shutil
+from sys import version_info
+
+from avocado import Test
+from avocado import skipIf
+from avocado.utils import process, build, memory, archive
+from avocado.utils.software_manager.manager import SoftwareManager
+
+SINGLE_NODE = len(memory.numa_nodes_with_memory()) < 2
+VERSION_CHK = version_info[0] < 4 and version_info[1] < 7
+
+
+class WillItScaleTest(Test):
+    """
+    Will It Scale takes a testcase and runs n parallel copies to see if the
+    testcase will scale.
+    Source - https://github.com/antonblanchard/will-it-scale
+
+    :avocado: tags=kernel,ppc64le
+    """
+
+    def setUp(self):
+        """
+        To execute test using git copy
+          make
+          ./runalltests
+
+        To generate graphical results
+          ./postprocess.py
+        """
+        smm = SoftwareManager()
+        pkgs = ['gcc', 'make', 'hwloc-devel']
```

Comment:
is `hwloc-devel` package available on all distro , as i am locate this only in RHEL9.*/centos9 onward ? Please double-check and consider test case in other distro if test not supported or we can have other alternative of this package ,

Reply:
@sacsant thanks for addressing review comments, can you please look in this comment as well

Reply:
Ah, thanks for the reminder. I missed this one. I checked RHEL & Ubuntu and the package is available. SLES does not seem to have this package. I can add a check for distro versions but was wondering since the test gets cancelled anyway if the package is not available, should we add this extra code?